test/test_nestedtensor.py

```diff
@@ -499,13 +499,15 @@ def test_zero_(self):
             t.fill_(0.)
             self.assertEqual(nt_ub, t)
 
-    @parametrize("func", [torch.ones_like, torch.zeros_like],
```

Comment:
Note: This currently passes, but I can imagine an alternative implementation of `randn()` for NT that doesn't have the property of "generating all random numbers at once -> gives same numbers as iteratively generating numbers matching underlying component sizes", and it could be argued this is no less correct. So I'm open to better forms of testing.